src/Livewire/Order/AdditionalAddresses.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace FluxErp\Livewire\Order;
+
+use FluxErp\Livewire\Forms\OrderAddressesForm;
+use Illuminate\Validation\ValidationException;
+use Illuminate\View\View;
+use Livewire\Attributes\Lazy;
+use Livewire\Component;
+use WireUi\Traits\Actions;
+
+#[Lazy]
+class AdditionalAddresses extends Component
+{
+    use Actions;
+
+    public int $orderId;
+
+    public OrderAddressesForm $form;
+
+    public ?int $address_id = null;
+
+    public ?int $address_type_id = null;
+
+    public function render(): View
+    {
+        $this->form->id = $this->orderId;
+        $this->form->addresses = resolve_static(\FluxErp\Models\Order::class, 'query')
+            ->whereKey($this->orderId)
+            ->with('addresses')
+            ->first(['id'])
+            ->addresses
+            ->map(fn ($address) => [
+                'address_type' => $address->pivot->addressType->name,
+                'address_id' => $address->id,
+                'address' => $address->postal_address,
+                'address_type_id' => $address->pivot->address_type_id,
+                'type' => $address->pivot->addressType->name,
+            ])
+            ->toArray();
+
+        return view('flux::livewire.order.additional-addresses');
+    }
+
+    public function placeholder(): View
+    {
+        return view('flux::livewire.placeholders.box');
+    }
+
+    public function delete(int $id): void
+    {
+        $this->form->addresses = array_filter(
+            $this->form->addresses,
+            fn (array $address) => $address['address_id'] !== $id
+        );
+
+        try {
+            $this->form->save();
+        } catch (ValidationException $e) {
+            exception_to_notifications($e, $this);
```

Comment:
revert removal of given address_id if validation failed.

Reply:
happens in line 78